.changeset/lemon-vans-film.md

```diff
@@ -0,0 +1,9 @@
+---
+"wrangler": minor
```

Comment:
This is a breaking change, and would have to be a `major` bump.

We can probably bundle this into v4, but in the meantime could you update this PR to deprecate this command instead of removing it (i.e. preserve the functionality but print a warning)